model/attention.py

```diff
@@ -3,15 +3,15 @@
 import torch.nn.functional as F
 
 class Attention(nn.Module):
-	def __init__(self, image_features_dim, decoder_hidden_state_dim, attention_dim):
+	def __init__(self, image_features_dim, decoder_hidden_state_dim, attention_dim, device):
```

Comment:
I am not sure about it but you are already doing the .to(device) when you create the model in the main.py, so I don't think it's necessary here, but check it just in case